domains/vollow.json

```diff
@@ -1,9 +1,12 @@
 {
   "owner": {
     "username": "vollowx",
-    "email": "muffinslog@proton.me"
+    "email": "vollowx@proton.me"
   },
   "record": {
-    "CNAME": "vollowx.github.io"
+    "MX": [
+      "dion.ns.cloudflare.com",
+      "treasure.ns.cloudflare.com"
+    ]
```

Comment:
Friendly reminder that this is the wrong way to add NS records + We don't support them currently.